src/main/java/edu/harvard/iq/dataverse/api/Info.java

```diff
@@ -139,6 +142,20 @@ public Response getExportFormats() {
         return ok(responseModel);
     }
 
+    @GET
+    @Path("settings/customization/{customizationFileType}")
+    public Response getCustomizationFile(@PathParam("customizationFileType") String customizationFileType) {
+        StringWriter stringWriter = new StringWriter();
+        PrintWriter printWriter = new PrintWriter(stringWriter);
+        customizationFilesService.getContents(printWriter, customizationFileType);
+        String contents = stringWriter.toString();
+        if (contents != null && !contents.isEmpty()) {
+            return ok(contents);
```

Comment:
Instead of returning...

```
{
    "status": "OK",
    "data": {
        "message": "<!doctype html>\n<html...
```

... (JSON, that is), should we consider returning the HTML file itself? Or the logo itself? The CSS file itself? And the proper content type, of course. This is what the download API does.